DataStructure/src/main/list/ResizableArray.java

```diff
@@ -100,10 +111,23 @@ public int removeAtIndex(int index) {
      * @return true when given element is removed, false otherwise.
      */
     public boolean remove(int num){
-        int index = indexOf(num);
-        if (index == -1) {return false;}
-        removeAtIndex(index);
-        return true;
+        if (contains(num)) {
```

Comment:
Inefficient code. Calling contains and then removing given number. Two passes on the list. It could be done in one pass.